test/test_cpp_extensions.py

```diff
@@ -141,6 +145,93 @@ def test_jit_cuda_extension(self):
         # 2 * sigmoid(0) = 2 * 0.5 = 1
         self.assertEqual(z, torch.ones_like(z))
 
+    def _run_jit_cuda_archflags(self, flags, expected):
+        # Compile an extension with given `flags`
+        def _check_cuobjdump_output(expected_values, is_ptx=False):
+            elf_or_ptx = '--list-ptx' if is_ptx else '--list-elf'
+            # Note, .extension name may include _v1, _v2, so first find exact name
+            ext_filename = glob.glob(os.path.join(temp_dir, 'cudaext_archflag*'))[0]
```

Comment:
Are you sure this one selected the expected file? I ask this because I saw this line `Output: cuobjdump info    : File 'C:\Windows\TEMP\tmpn5dwlrij\cudaext_archflags.exp' does not contain device code` in the log. But in Windows, `.exp` file is only a file that contains the symbol table, the object file has the suffix `.obj`.

Reply:
ah, staring for too long at similar build log - apologies for the noise. I'll put back the bit that selects the `.pyd` file only on Windows.